{
  try {
    Object result=super.get();
    callback.onResponse(result);
    return result;
  }
 catch (  Throwable t) {
    callback.onFailure(t);
    return ExceptionUtil.sneakyThrow(t);
  }
}
